index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <nav class="nav-bar">
+            <div class = "ham-menu">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+           </div>
+        </nav>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
+        <div class="title-name">
+            <h2>I’m Amol
+                Glad to see you!
+            </h2>
```

Comment:
- [ ] As requested by the previous reviewer, kindly change the `<h2>` tag here to a `<h1>` tag, as it is the main title of your page.

Reply:
Implemented 👍